tests/components/hunterdouglas_powerview/conftest.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for the Hunter Douglas PowerView integration."""
+import json
+from unittest.mock import patch
+
+import pytest
+
+from tests.common import load_fixture
+
+
+@pytest.fixture
+def mock_powerview_v2_hub():
+    """Mock a Powerview v2 hub."""
+    userdata = json.loads(load_fixture("hunterdouglas_powerview/userdata.json"))
+    fwversion = json.loads(load_fixture("hunterdouglas_powerview/fwversion.json"))
+    scenes = json.loads(load_fixture("hunterdouglas_powerview/scenes.json"))
```

Comment:
Would it make sense to make these session fixtures so the JSON is only loaded once in case this fixture is used multiple times ?

Reply:
good idea

done in 8c0cad3a9ec